components/automate-ui/Makefile

```diff
@@ -41,12 +41,12 @@ lint-typescript:
 
 #: Security validation via semgrep
 semgrep:
-	semgrep --config $(REPOROOT)/semgrep
-	semgrep --config https://semgrep.dev/p/r2c-ci --autofix
```

Comment:
Autofix here was a mistake; that is just supposed to be in the `semgrep-and-fix` target.